third_party_license_file_generator/site_packages.py

```diff
@@ -22,8 +22,7 @@ def _pre_exec():
 def _run_subprocess(command_line):
     if platform.system() == 'Windows':
         p = subprocess.Popen(
-            executable=sys.executable,
-            args=shlex.split(command_line),
+            args=command_line.split(),
```

Comment:
Is there a reason to change from `shlex.split()` to `string.split()`? I don't know of a case where `string.split()` fails, but `shlex.split()` [appears](https://code-maven.com/split-command-line-into-pieces) to be more robust to whitespace.